src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -37,6 +39,24 @@ def list(self):
         fsmap = mgr.get("fs_map")
         return fsmap['filesystems']
 
+    def create(self, name: str, service_spec: Dict[str, Any]):
+        service_spec_str = '1 '
```

Comment:
Can be made much simpler with `.join()`
```suggestion
        labels_str = ','.join(f'label:{label}' for label in placement.get('labels', []))
        hosts_str = ','.join(placement.get('hosts', []))
       
       service_spec_str = f"1 {labels_str}{',' if labels_str and hosts_str else ''}{hosts_str}"
```